scripts/rnnlm/initialize_matrix.pl

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env perl
+# Copyright 2017     Hossein Hadian
```

Comment:
as far as I can see from github, you didn't set the mode right.  there is a special invocation for when you forgot to give it the right mode before git add-ing it.  look it up.